backend/open_webui/routers/artifacts.py

```
@@ -305,15 +308,65 @@ async def update_redaction_config(
+                        "success": False,
+                        "detail": "Configuration validation failed",
+                        "errors": issues,
+                    },
+                )
+
+            updated_state = config.export_config()
+
+        try:
+            save_config_to_storage()
+        except Exception as exc:
+            log.error("Failed to persist redaction configuration: %s", exc)
```

Ruff (0.13.1)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)